Vienna/Sources/Main window/WebKitArticleTab.swift

```diff
@@ -59,6 +59,17 @@ class WebKitArticleTab: BrowserTab, ArticleContentView, CustomWKUIDelegate {
     init() {
         super.init()
         self.webView.contextMenuProvider = self
+        self.registerNavigationEndHandler { [weak self] _ in self?.deleteHtmlFiles() }
+    }
+
+    func deleteHtmlFiles() {
```

Comment:
The function name is not very adequate: it deletes a single file, not multiple ones…